Backend/app/rider-platform/rider-app/Main/package.yaml

```
@@ -206,4 +206,25 @@ executables:
+    dependencies:
+      - rider-app
+      - mobility-core
+      - tasty
+      - tasty-hunit
+      - tasty-quickcheck
+      - safe-exceptions
+      - base64-bytestring
+      - cryptonite
+      - time
+      - text
+      - bytestring
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)